### ISSUE TITLE
Block indent_style for attributes?

### ISSUE THREAD

**Issue by behnam** [rust-lang-nursery/rustfmt#2423]
Is there a config to let me change the default formatting of

```rust
#![forbid(bad_style, future_incompatible, missing_debug_implementations, missing_docs,
          unconditional_recursion, unsafe_code, unused, unused_imports)]
```

to

```rust
#![forbid(
    bad_style,
    future_incompatible,
    missing_debug_implementations,
    missing_docs,
    unconditional_recursion,
    unsafe_code,
    unused,
    unused_imports,
)]
```
?

Apparently `indent_style` does not apply here, and there's no single reference to *attributes* in <Configurations.md>.

**Comment by topecongiro** [comment]
In [the RFC for attribute](https://github.com/rust-lang-nursery/fmt-rfcs/blob/master/guide/guide.md#attributes):

> For attributes with argument lists, format like functions.

so probably we should use the block indent by default, and let `indent_style` config option apply to attributes as well? Though AFAIK there has not been much discussion on how to format attributes with argument lists.

**Comment by topecongiro** [comment]
Closed via #2809.
